src/System Application/App/File System/src/Lookup/FileType.Enum.al

```diff
@@ -0,0 +1,31 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+/// <summary>
+/// Indicator of what type the resource is.
+/// </summary>
+enum 9452 "File Type"
+{
+    Access = Public;
+    Extensible = false;
+
+    /// <summary>
+    ///  Indicates if entry is a directory.
```

Comment:
Those Captions shouldn't be locked, since this Enum is used on a page (`page 9455 "File Account Browser"`)

Reply:
Makes total sense. Is solved.